src/components/shared/Slider/Slider.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const Slider = ({ min, max, step, defaultValue, handleChange }) => {
```

Comment:
Since we will use it with a predefined set of values, what do you think about receiving the set of values in place of min, max, and step?

```suggestion
export const Slider = ({ values, defaultValue, handleChange }) => {
```

Reply:
In this case, I am not sure how default values would work.

Does it work?

```suggestion
export const Slider = ({ values, defaultValue = values?.lenght && values[0], handleChange }) => {
```

Reply:
Or, if you want to make honor to the original API, what do you think about simply pass the parameters?

```suggestion
export const Slider = (props) => (<input 
      style={{ marginTop: '100px', padding: '0' }}
      type="range"
      {...props}
      // . . .
```

Reply:
By the moment I won't apply any change here.